src/main/java/org/itsallcode/openfasttrace/importer/pdf/PdfFileImporter.java

```diff
@@ -0,0 +1,107 @@
+package org.itsallcode.openfasttrace.importer.pdf;
+
+import java.io.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.text.PDFTextStripper;
+import org.itsallcode.openfasttrace.core.SpecificationItemId;
+/*-
+ * #%L
+ * OpenFastTrace
+ * %%
+ * Copyright (C) 2016 - 2018 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+import org.itsallcode.openfasttrace.importer.*;
+import org.itsallcode.openfasttrace.importer.input.InputFile;
+
+/**
+ * This {@link Importer} supports reading PDF files
+ */
+public class PdfFileImporter implements Importer
+{
+    final static String DEFAULT_SPECITEM_PATTERN = "\\[(\\w+-\\d+)\\](.*)";
```

Comment:
Using non-standard specification item ID notation makes manual debugging harder when you want to copy-paste IDs from a trace report. I therefore recommend to use standard IDs.

Reply:
Sorry I didn't get this comment.
What do you mean by "non-standard"? This is the Requirement ID format of one of the specifications I want to import. There are others as well which are completely different.
Also I understand from the user guide that specification item names can contain letters, numbers, hyphen, underscore and dot, which should be ensured with this pattern.

Reply:
After talking to @kaklakariada I see clearer what the reason for the PDF importer is: a spec you get as delivery from a 3rd party. In that case forget my comment since it is clear that you have to live with what the delivery contains.